shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-api/src/main/java/org/apache/shardingsphere/encrypt/api/config/rule/EncryptColumnRuleConfiguration.java

```diff
@@ -23,17 +23,29 @@
 /**
  * Encrypt column rule configuration.
  */
-@RequiredArgsConstructor
 @Getter
+@RequiredArgsConstructor
```

Comment:
Please change the sequence, we'd like use first line for Constructor, and second line for getter and setter

Reply:
Thank you for reminding me. I have solved it.